cli/CHANGELOG.md

```diff
@@ -1,4 +1,13 @@
  <!-- See the ../guides/writing-the-cypress-changelog.md for details on writing the changelog. -->
+
+## 12.5.0
+
+_Released 01/31/2023_
+
+**Features:**
+
+- Easily debug failed CI test runs from within the Cypress app with the new "Debug" page. For more details, see the [Debug documentation](https://on.cypress.io/debug-page).
```

Comment:
IMO `Easily debug failed CI test runs` is misleading because we only show data if the run was recorded to the cloud. anyway to expand to indicate this is for recorded run?

Will there be a blog post associated with this feature release?

Reply:
@emilyrohrbough How about...

```suggestion
- Easily debug failed CI test runs from Cypress Cloud within the Cypress app with the new "Debug" page. For more details, see the [Debug documentation](https://on.cypress.io/debug-page).
```

Reply:
@warrensplayer Sounds good 👍🏻

Reply:
Maybe something like this

```suggestion
- Easily debug failed CI test runs right from your local Cypress app with the new Debug page, powered by Cypress Cloud. For more details, see the [Debug documentation](https://on.cypress.io/debug-page).
```

Reply:
I went with Peter's copy, removed the sub-bullets and referenced the PR instead of all the separate issues.